index.html

```diff
@@ -125,6 +129,299 @@ <h1>I'm Moise Glad to see you !</h1>
 					</li>
 				</ul>
 			</section>
+			<section class="works-section">
+				<ul class="works">
+					<li class="card-work">
+						<a class="portfolio-img" href="#">
+							<img src="images/Portfolio_1.png" alt="tonic" />
+						</a>
+						<div class="title">
+							<h2>Tonic</h2>
+						</div>
+						<div class="info">
+							<span>CANOPY</span>
+							<span><img src="images/Counter.png" alt="" /></span>
+							<span>Backend Dev</span>
+							<span><img src="images/Counter.png" alt="" /></span>
+							<span>2015</span>
+						</div>
+						<div class="detail">
+							<p>
+								A daily selection of privately personalized reads; no accounts
+								or sign-ups required.
+							</p>
+						</div>
+						<ul class="languages">
+							<li><a href="#">Html</a></li>
+							<li><a href="#">Css</a></li>
+							<li><a href="#">JavaScript</a></li>
+						</ul>
+						<div class="see-project">
+							<a class="btn" href="#">See Project</a>
+						</div>
+					</li>
```

Comment:
I added the right path for my images and I used UL instead of DIV tags within the works section. 👨‍💻💻😊